src/Collection/ProcessArgumentsCollection.php

```diff
@@ -31,10 +28,9 @@ public function addOptionalArgument($argument, $value)
     }
 
     /**
-     * @param string $argument
-     * @param string $value
+     * @param string|int|null $value
```

Comment:
Where did you get the list of supported types? It can accept any type that can be converted to string.